### ISSUE TITLE
Problem with some special characters

### ISSUE THREAD

**Issue by duongcat** [soldair/node-qrcode#104]
Hi,
I use function "toFile" to generate QR code, input string have special characters like dot (.) commna (,)
When I read QR code, I receive an other character
dot(.) --> -
comma (,) --> :

Please help me fix it!

**Comment by vigreco** [comment]
Hi @duongcat, 
thanks for reporting the issue.
I'll look into this.

**Comment by vigreco** [comment]
Ok, I found the problem!
There's a bug in a RegExp. I'll fix it by today.

Meanwhile, as a quick workaround, you can force the encoding to `byte`.
```javascript
QRCode.toFile(filename, [{ data: 'your text', mode: 'byte' }], function () {
    ...
}
```

Thanks! 😃

**Comment by vigreco** [comment]
Fixed in `v1.0.1`.
